### DISCUSSION TITLE
how can I run guage in WSL

### DISCUSSION THREAD

**Discussion by ifeue** [getgauge/gauge#2096] [General]
gauge + python3.8

gauge plugin: python, html-report, screenshot

gauge and it's plugins are up-to-date.

When run gauge in WSL2, got error as below:

Python: 3.8.10
Error occurred while waiting for runner process to finish.
Error : exit status 1

  Error Message: error reading from server: EOF
  Stacktrace:

  Error Message: Runner is not Alive
  Stacktrace:

Successfully generated html-report to => /home/ifeue/gitrepo/webdriverife/ife_project/reports/html-report/2021-09-02 09.17.10/index.html

Failed to kill Runner: rpc error: code = Unavailable desc = connection error: desc = "transport: Error while dialing dial tcp 127.0.0.1:43237: connect: connection refused"
Specifications: 0 executed      0 passed        0 failed        0 skipped
Scenarios:      0 executed      0 passed        0 failed        0 skipped

Total time taken: 201ms

**Comment by ifeue** [comment]
It works now. This is an error in my script.

**Reply by sankarasai18-gauge** [reply]
@ifeue if you don't mind, could you please give me background about the fix whichn you did on your script. I am facing same issue (https://github.com/getgauge/gauge/issues/2165)
